keyboards/tweetydabird/chameleon/info.json

```diff
@@ -0,0 +1,117 @@
+{
+    "manufacturer": "Tweetys Wild Thinking",
+    "keyboard_name": "Chameleon (QMK)",
+    "maintainer": "TweetyDaBird",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP18", "pin_b": "GP19", "resolution": 2},
+            {"pin_a": "GP27", "pin_b": "GP0", "resolution": 2},
+            {"pin_a": "GP17", "pin_b": "GP16", "resolution": 2},
+            {"pin_a": "GP10", "pin_b": "GP9", "resolution": 2}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true,
+        "tri_layer": true
+    },
+    "matrix_pins": {
+        "cols": ["GP15", "GP14", "GP6", "GP5", "GP4", "GP3"],
+        "rows": ["GP29", "GP22", "GP21", "GP24", "GP13", "GP1", "GP11", "GP2", "GP12", "GP8"]
+    },
+    "processor": "RP2040",
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "rainbow_mood": true
+        },
+        "layers": {
+            "enabled": true
+        },
+        "led_count": 8,
+        "sleep": true
+    },
+    "url": "https://lectronz.com/stores/tweetys-wild-thinking",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x23B0",
+        "vid": "0xFEED"
+    },
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP20"
+    },
+    "layouts": {
```

Comment:
could the following Community Layouts (`layouts/default`) be added?:
- orthro_4x12
- planck_mit

Reply:
Yeah... As soon as I find a manual to doing all that correctly. The QMK doc's are a bit sparse...

Reply:
Let's try this... I have no clue!